### DISCUSSION TITLE
Daily 17/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#156] [General]
El domingo durante el día estuve haciendo algunos arreglos simples de semántica en los html, y al final hice el merge algunas horas antes del límite. Armé la discusión sobre la weekly. También hice la lección de inglés semanal que me había olvidado. Hoy probablemente haga la siguiente lección de inglés y por ahí empiece a hacer las user stories y las issues que necesitemos en el backlog. También debería actualizar el pdf con respecto a los sprint para el próximo merge a main.

**Comment by AhumadaCeleste** [comment]
Ayer, pude terminar con Css, vi la clases del jueves pasado de Gestión de contenido, y comencé a hacer pruebas en Wordpress siguiendo la clase del profe. También comencé estudiar el apunte de wordpress, lo referido a la historia del mismo y termine de estudiar lo del examen de Proyecto. Solo me estaría faltando lo nuevo de Git, para poder repasar todo con tiempo durante la semana ya que contare con muy poco tiempo por el trabajo. Hoy veré la clase del jueves de FullStack. También, estoy intentando armar un estilo para nuestra página en wordpress al cual ya le asigne incluso un eslogan para proponerlo a concentración del grupo. Aún no conozco java pero investigare un poco para pensar en ideas aplicables según lo propuesto en la reunión del domingo. Mañana quisiera comenzar con boobstarp y avanzar con los vídeos del Profe Mainero.

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve haciendo algunas tareas que  habian quedado pendientes para entregar en el sprint1. Participé de la Planning y Review del domingo.
Hoy realicé el script de BD que agregaron en el día de la fecha para el sprint 1. Estaré estudiando para la evalución de Gestión de proyectos y poniendome al día con Inglés. Espero poder avanzar con estos temas para poder dedicarle tiempo a Wordpress en  la semana
